pkg/apis/providerconfig/v1alpha1/types.go

```diff
@@ -109,10 +65,6 @@ type CRUDConfig struct {
 	// handling machine Delete operations (power-off/deprovisioning)
 	DeleteArgs []string `json:"deleteArgs,omitempty"`
 
-	// Optional command to be used instead of the default when
```

Comment:
Why?

Reply:
Again I prefer do not assume what we will need in the future, also in this case if you will need it, you will only re-introduce it. But I prefer do not overload struct with fields that we currently do not use.

Reply:
Re-introducing later something we know now that we will need is an unnecessary API change. Please undo.

Reply:
Can you give me some proof that it will be unnecessary API change in the near future?

Reply:
We absolutely need support for reboots in some form, so I just figured we might as well add the field now rather than create a new version of the API.